bait-files/src/main/java/com/google/sps/servlets/Bootstrapper.java

```diff
@@ -1,5 +1,9 @@
 package com.google.sps.servlets;
-import com.google.sps.servlets.UserAccessor;
+import com.google.appengine.api.blobstore.BlobInfo;
+import com.google.appengine.api.blobstore.BlobInfoFactory;
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
```

Comment:
Do you need any of these imports?
(I'm somewhat blind, so maybe the answer is "yes" haha.  I just don't see them used in this class.)

Reply:
removed imports